### PR TITLE
Feature/add anchorclick option

### PR DESCRIPTION
**Pull request by nathanbedford**

### CODE REVIEW

**Comment by nathanbedford** [comment]
@xStrom @jmeas @reppners any chance you've got some time to review this?

**Comment by xStrom** [comment]
There seem to be merge conflicts. Could you update your commit?

**Comment by nathanbedford** [comment]
Shoot, my mistake. i just merged a PR and that might have led to the conflict. I will fix once I'm at my computer.

**Comment by nathanbedford** [comment]
@xStrom merged conflict resolved

**Comment by nathanbedford** [comment]
@xStrom do you have a few minutes to check this out now that it's mergable?

File: ios-drag-drop.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+  
+function _exposeIosHtml5DragDropShim(config) {
+  var shimConfig;
   log = noop; // noOp, remove this line to enable debugging
 
   var coordinateSystemForElementFromPoint;
 
   function main() {
     config = config || {};
+    shimConfig = config;
```
Comment (xStrom) [inline ios-drag-drop.js]: What's the purpose of this `shimConfig` variable? Looks to me that we could just check `config` directly like we do for other options.

Comment (nathanbedford) [inline ios-drag-drop.js]: You are right...this config object wasn't available to all places of the shim before.  I'll update this now.

**Comment by nathanbedford** [comment]
RE: " So the general idea of this change is to treat all anchors as if they had an explicit draggable="false"?"

The current default behavior is that, when a user starts scrolling by touching a link (a tag) and then scrolling, the shim simulates a mouse click.  The reference given for this behavior is [here](https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Touch_events#Handling_clicks).

This PR gives us a way to opt out of that default behavior, because I don't really want to navigate to a link just because I started a scroll motion on it.

**Comment by xStrom** [comment]
Yeah, the current anchor click simulation is really basic. I know, because I wrote it and it has never improved because of the 2.0 rewrite branch.

What I'm really getting at here is - do you want to only stop the click simulation, or do you also want to stop the drag event (like this PR currently stops)?

**Comment by nathanbedford** [comment]
Ah, got it now...yeah, you found a bug in my PR.  I'm fixing now.  I didn't want to prevent the drag event, I just wanted to prevent the click simulation.

I didn't realize you wrote the original implementation...I'm glad your on this PR!  Thanks for the assist.

**Comment by nathanbedford** [comment]
@xStrom I've addressed the issues you noticed.

**Comment by xStrom** [comment]
OK, let's do some bikeshedding over the option naming. I feel something like `config.simulateAnchorClick` would more accurately describe what the option does. Thoughts?

If we were to change it, then we'd have to default to `true` on it. Perhaps by adding a line into `main()`.

```javascript
config = config || {};
if (!config.hasOwnProperty("simulateAnchorClick")) config.simulateAnchorClick = true;
```

**Comment by nathanbedford** [comment]
I agree.  That's more readable.  Fixing now.

**Comment by nathanbedford** [comment]
Updated property name, updated related logic and readme

**Comment by nathanbedford** [comment]
Thanks for the help, @xStrom

**Pull request merged**